guides/release/accessibility/components.md

```diff
@@ -1,89 +1,88 @@
 
-## Focus management in components
+## Gestion du focus dans les composants
 
-Focus is one of the main ways a component can communicate with screen readers.
+Le focus est une manière parmi tant d'autres pour un composant de communiquer avec les lecteurs d'écrans.
 
-For example, when you hit tab on a page or click on a form field, a blue border usually appears around the element. This kind of behavior is part of focus.
-Developers can use JavaScript to control the focus in their apps, enabling keyboard navigation and usability by screen readers.
-For example, if there is a button that launches a modal with interactive elements in it, that button's click handler needs to contain code that brings focus to the new content.
+Par exemple quand vous pressez l'onglet d'une page, ou cliquez dansun champ de formulaire, une bordure bleue va le plus souvent apparaîtree autour de l'élément. C'est en partie ce que l'on peut appeler le focus.
+Les développeurs.euses peuvent utiliser JavaScript pour contrôler le focus dans leurs applications, permettant la navigation au clavier, et l'utilisation des lecteurs d'écrans.
 
-This article is a good launching point for learning more about focus: [Keyboard accessibility](https://webaim.org/techniques/keyboard/)
+Cet article est une bonne introduction pour en apprendre plus sur le focus: [Keyboard accessibility](https://webaim.org/techniques/keyboard/)
 
-Here are some other tips to get you started:
+Voici quelques astuces pour bien commencer:
 
-- There is a difference between browse mode and focus mode in screen readers- see ["Focus Please"](https://codepen.io/melsumner/live/ZJeYoP).
-- Focus should return from whence it came- for example, if a `<button>` element opens a modal, the focus should then be returned to that same trigger button once the modal is closed. 
-- Note: `role="presentation"` or `aria-hidden="true"` should not be used on a focusable element.
+- Il y a une différence entre le mode navigateur et le mode focus dans un lecteur d'écrans- En savoir plus: ["Focus Please"](https://codepen.io/melsumner/live/ZJeYoP).
+- Le focus doit retourner d'où il provient, par exemple, si un élément bouton `<button>` ouvre une modale, le focus devrait ensuite retourner sur le même élément bouton quand cette modale est fermée.
+- On notera que `role="presentation"` ou `aria-hidden="true"` ne devraient pas être utilisés sur un élément focalisable.
 
+## Nom accessible
 
-## Accessible name
+Tous les éléments interactifs doivent avoir un nom accessible. Mais qu'est ce que ça signifie exactement?
 
-All interactive elements must have an accessible name. But what does that mean, exactly? 
+Parce que le code fourni doit être lisible par d'autres machines (les technologies d'assistance comme les lecteurs d'écrans par exemple). Il y a également de la documentation sur comment on détermine un nom accessible: [Accessible Name and Description Computation](https://www.w3.org/TR/accname-1.1/).
```

Comment:
```suggestion
Du fait que le code fourni doit être lisible par d'autres machines (les technologies d'assistance comme les lecteurs d'écrans par exemple), voici de la documentation sur comment un nom accessible est déterminé : [Accessible Name and Description Computation](https://www.w3.org/TR/accname-1.1/).
```

Reply:
Considering the previous sentence, I would rather go for something more custom here:
```suggestion
Ça signifie que le code fourni doit être lisible par d'autres machines (par exemple, les technologies d'assistance comme les lecteurs d'écrans). Voici de la documentation pour comprendre comment ce nom accessible est déterminé: [Accessible Name and Description Computation](https://www.w3.org/TR/accname-1.1/).
```